dask/tests/test_sharedict.py

```diff
@@ -1,15 +1,28 @@
+import sys
+import warnings
 import pytest
 from toolz import merge
 
 from dask.compatibility import Mapping
-from dask.sharedict import ShareDict
+
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore", UserWarning)
+    from dask.sharedict import ShareDict
 
 
 a = {'x': 1, 'y': 2}
 b = {'z': 3}
 c = {'w': 2}
 
 
+def test_import_warns():
```

Comment:
Hopefully this test is robust to all the import caching that could occur.

Reply:
Hmm, seems like it's not robust... I'm just going to assume that we are doing things correctly and not test this.